tests/bluetooth/bsim_bt/bsim_test_host/src/main_eatt.c

```diff
@@ -0,0 +1,488 @@
+/* main.c - Application main entry point */
```

Comment:
The test does not actually test the bearer used, but it is logged on the server side.

Reply:
Output from test:
```
d_00: @00:00:00.000000  *** Booting Zephyr OS build v2.7.99-3159-ga171b19c0d90  ***
d_00: @00:00:00.000000  [00:00:00.000,000] <inf> bt_hci_core: HW Platform: Nordic Semiconductor (0x0002)
d_01: @00:00:00.000000  *** Booting Zephyr OS build v2.7.99-3159-ga171b19c0d90  ***
d_00: @00:00:00.000000  [00:00:00.000,000] <inf> bt_hci_core: HW Variant: nRF52x (0x0002)
d_00: @00:00:00.000000  [00:00:00.000,000] <inf> bt_hci_core: Firmware: Standard Bluetooth controller (0x00) Version 2.7 Build 99
d_00: @00:00:00.000000  [00:00:00.000,000] <wrn> bt_id: No static addresses stored in controller
d_01: @00:00:00.000000  [00:00:00.000,000] <inf> bt_hci_core: HW Platform: Nordic Semiconductor (0x0002)
d_01: @00:00:00.000000  [00:00:00.000,000] <inf> bt_hci_core: HW Variant: nRF52x (0x0002)
d_01: @00:00:00.000000  [00:00:00.000,000] <inf> bt_hci_core: Firmware: Standard Bluetooth controller (0x00) Version 2.7 Build 99
d_01: @00:00:00.000000  [00:00:00.000,000] <wrn> bt_id: No static addresses stored in controller
d_00: @00:00:00.002648  [00:00:00.002,624] <inf> bt_hci_core: Identity: ED:3B:20:15:18:12 (random)
d_00: @00:00:00.002648  [00:00:00.002,624] <inf> bt_hci_core: HCI: version 5.3 (0x0c) revision 0x0000, manufacturer 0x05f1
d_00: @00:00:00.002648  [00:00:00.002,624] <inf> bt_hci_core: LMP: version 5.3 (0x0c) subver 0xffff
d_01: @00:00:00.002648  [00:00:00.002,624] <inf> bt_hci_core: Identity: ED:71:8F:C2:E4:6E (random)
d_01: @00:00:00.002648  [00:00:00.002,624] <inf> bt_hci_core: HCI: version 5.3 (0x0c) revision 0x0000, manufacturer 0x05f1
d_01: @00:00:00.002648  [00:00:00.002,624] <inf> bt_hci_core: LMP: version 5.3 (0x0c) subver 0xffff
d_00: @00:00:00.318322  Device connected
d_01: @00:00:00.426151  Connected: ED:3B:20:15:18:12 (random)
d_00: @00:00:00.426148  Connected: ED:71:8F:C2:E4:6E (random)
d_00: @00:00:00.439087  Connecting 1 EATT channels
d_00: @00:00:01.886078  char_1_attr_handle: 20
d_00: @00:00:01.886078  char_2_attr_handle: 22
d_00: @00:00:01.886078  Reading with flag BT_ATT_BEARER_UNENHANCED
d_01: @00:00:01.927921  [00:00:01.927,917] <inf> bt_att: Receiived on enhanced bearer? 0
d_01: @00:00:01.927921  read_char_1 bt_gatt_attr_read returned 1500
d_00: @00:00:02.006565  gatt_read_1_cb: read data: 0, length: 1500, err: 0x0
d_00: @00:00:02.006565  gatt_read_1_cb: read data: 1, length: 0, err: 0x0
d_01: @00:00:02.027486  [00:00:02.027,465] <inf> bt_att: Receiived on enhanced bearer? 0
d_01: @00:00:02.027486  read_char_2 bt_gatt_attr_read returned 10
d_00: @00:00:02.077714  gatt_read_2_cb: read data: 0, length: 10, err: 0x0
d_00: @00:00:02.077714  gatt_read_2_cb: read data: 1, length: 0, err: 0x0
d_00: @00:00:11.875306  Reads done
d_00: @00:00:11.875306  Reading with flag BT_ATT_BEARER_ENHANCED
d_01: @00:00:11.877525  [00:00:11.877,502] <inf> bt_att: Receiived on enhanced bearer? 1
d_01: @00:00:11.877525  read_char_1 bt_gatt_attr_read returned 1500
d_00: @00:00:11.956991  gatt_read_1_cb: read data: 2, length: 1500, err: 0x0
d_00: @00:00:11.956991  gatt_read_1_cb: read data: 3, length: 0, err: 0x0
d_01: @00:00:11.977524  [00:00:11.977,508] <inf> bt_att: Receiived on enhanced bearer? 1
d_01: @00:00:11.977524  read_char_2 bt_gatt_attr_read returned 10
d_00: @00:00:11.978222  gatt_read_2_cb: read data: 2, length: 10, err: 0x0
d_00: @00:00:11.978222  gatt_read_2_cb: read data: 3, length: 0, err: 0x0
d_00: @00:00:21.864533  Reads done
d_00: @00:00:21.864533  Reading with flag BT_ATT_BEARER_ANY
d_01: @00:00:21.877525  [00:00:21.877,502] <inf> bt_att: Receiived on enhanced bearer? 1
d_01: @00:00:21.877525  read_char_1 bt_gatt_attr_read returned 1500
d_01: @00:00:21.877941  [00:00:21.877,929] <inf> bt_att: Receiived on enhanced bearer? 0
d_01: @00:00:21.877941  read_char_2 bt_gatt_attr_read returned 10
d_00: @00:00:21.907443  gatt_read_1_cb: read data: 4, length: 1500, err: 0x0
d_00: @00:00:21.907443  gatt_read_1_cb: read data: 5, length: 0, err: 0x0
d_00: @00:00:21.908376  gatt_read_2_cb: read data: 4, length: 10, err: 0x0
d_00: @00:00:21.908376  gatt_read_2_cb: read data: 5, length: 0, err: 0x0
d_00: @00:00:31.853760  Reads done
d_00: @00:01:41.327684  Disconnected: ED:71:8F:C2:E4:6E (random) (reason 0x13)
d_00: @00:01:41.339264 INFO: EATT Central tests Passed
d_01: @00:01:41.327685  Disconnected: ED:3B:20:15:18:12 (random) (reason 0x16)
d_01: @00:01:41.359223 INFO: EATT Peripheral tests Passed

```